### PR TITLE
chore: enabled cypress test in filter file

### PR DESCRIPTION
**Pull request by adam-stasiak**
### SUMMARY

I enabled test in dashboard/filter.test.ts. It was needed to change a "op" expected value. Added also timeout to wait for requests.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/cypress-base/cypress/integration/dashboard/filter.test.ts [hunk 1]
```diff
@@ -101,16 +101,19 @@ describe('Dashboard filter', () => {
     cy.get('.filter_box button').click({ force: true });
 
     // wait again after applied filters
-    cy.wait(aliases.filter(x => x !== getAlias(filterId))).then(requests => {
+    cy.wait(
+      aliases.filter(x => x !== getAlias(filterId)),
+      { timeout: 30000 },
```
Comment (rusackas) [inline superset-frontend/cypress-base/cypress/integration/dashboard/filter.test.ts:106]: This seems like a reeeeeally long timeout. Is there a reason for this? If so, maybe add a comment to the code so people don't follow the pattern here unnecessarily, and potentially slow down a hundred failing tests by 30 seconds each ;)

**Review by rusackas — APPROVED** [review]
Generally LGTM, though I added a question about the necessity of a long timeout.

File: superset-frontend/cypress-base/cypress/integration/dashboard/filter.test.ts (hunk 1 above)
Comment (adam-stasiak) [reply superset-frontend/cypress-base/cypress/integration/dashboard/filter.test.ts]: I found the problem is in typing propably, deleted timeout and set delay to "type" operation - delay between each key entered. Triggered run to verify this on CI.

Comment (adam-stasiak) [reply superset-frontend/cypress-base/cypress/integration/dashboard/filter.test.ts]: it still seems to be flaky... I will try to fix first failing assertion from latest pipeline

**Comment by adam-stasiak** [comment]
I modified logic of this test to use selection from list instead of typing which was quite unstable while I was testing it. Problem with this Dashboard is rendering consumes more time than requests -> while graphs are rendering and we start typing then Cypress cannot proceed with successfull typing.

**Review by rusackas — APPROVED** [review]
This is looking much better, thanks for the extra work on the speed/reliability!

**Pull request merged**